tests/test_abstract_repr.py

```diff
@@ -89,6 +96,34 @@
 )
 
 
+def test_abstract_repr_encoder_non_torch():
```

Comment:
You can move all these tests to `test_backend_abstract_repr.py` now

Reply:
even the one for the base encoder? The torch part I can see, but why the non-torch part? It's not backend specific.

Reply:
You're right, I meant just the tests for `Results`